packages/v4/src/content/design-guidelines/components/empty-state/empty-state.md

```diff
@@ -3,208 +3,393 @@ id: Empty state
-| <img src="./img/success.png" alt="success icon" width="200"/> |Success |Green (#486b00) |fa-check-circle |Use to indicate that a task or process has been completed successfully
-| <img src="./img/add.png" alt="add icon" width="200"/> |Addition or creation |Grey (#8A8D90) |fa-plus-circle |Use when something needs to be added or created
+<div class="ws-content-table">
+
+| **Do**                      | **Don't**                  |
+|:-------------------------------:|:--------------------------:|
+| Welcome to PatternFly | Welcome to PatternFly! |
+
+</div>
+<br /> 
+
+**Body**: Explain how the product or feature delivers value to the user. State what the user can do with the product instead of what the product enables them to do. This way, you give power to the user.
+
+<div class="ws-content-table">
+
+| **Do**                      | **Don't**                  |
+|:-------------------------------:|:--------------------------:|
+| Access all the resources you need to build scalable experiences in the open. | PatternFly enables you to build scalable experiences in the open. |
+
+</div>
+<br />  
+
+**Button**: Give the user the ability to easily get started with the product.  
+
+<div class="ws-content-table">
+
+| **Do**                      | **Don't**                  |
+|:-------------------------------:|:--------------------------:|
+| Get started | Exit |
+
+</div>
+<br />  
+
+### No results
+No results empty states appear when the system doesn’t find any data to show, such as when a user’s search criteria doesn’t yield any results.  
+
+In your empty state, explain that nothing was found and give the user a next step so that they can continue their search.  
+
+Use the small empty state variation for no results empty states. 
+
+<img src="./img/no-data.png" alt="empty state showing no search results" width="1500px"/>    
+
+**Heading**: Briefly state that no results were found.
+
+<div class="ws-content-table">
+
+| **Do**                      | **Don't**                  |
+|:-------------------------------:|:--------------------------:|
+| No results found | Your search did not yield any results |
+
+</div>
+<br /> 
+
+**Body**: Explain what the user should do next so that they can continue searching for what they need.
+
+<div class="ws-content-table">
+
+| **Do**                      | **Don't**                  |
+|:-------------------------------:|:--------------------------:|
+| Adjust your filters and try again. | We did not find any results that matched your filters. |
+
+</div>
+
+### Required configuration  
+Required configuration empty states appear when the user needs to configure, connect, or enable something before using a feature.  
+
+Use the large empty state variation for required configuration empty states.
+
+<img src="./img/configuration.png" alt="empty state with system configuration information" width="1500px"/>  
```

Comment:
Looks like this isn't the correct image - the one in the sketch file looks like this:
![image](https://user-images.githubusercontent.com/51165119/113754654-73d1cc00-96dd-11eb-80fb-744b22b050c1.png)

Reply:
It is from the Sketch file. I just had to change the microcopy so that the image shows the correct design _and_ the correct microcopy for the headline, body text, and button.

Reply:
Ah I see, well the changes in copy messed with the spacing again! Here's an update image with your wanted microcopy and the correct spacing :) I think after you make this change it will be good to merge, thanks Abi!
![configuration](https://user-images.githubusercontent.com/51165119/114081466-9816df80-987a-11eb-8807-49fe7c94bffc.png)